include/packet/radiotap_header.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright nPrint 2021
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ */
+
+#ifndef RADIOTAP_HEADER
+#define RADIOTAP_HEADER
+
+#include "packet_header.hpp"
+
+#define SIZE_RADIOTAP_HEADER_BITSTRING 56
+
+struct radiotap_header {
+    uint8_t  radiotap_data[56];	/* all bytes for radiotap_data 	*/
```

Comment:
Is there a reason we're allocating a buffer here when we actually take a pointer to a raw buffer for setting the raw data of a header?

Reply:
Good catch. We should use a pointer here, right?

Reply:
yes, just a pointer works here.

Reply:
Resolved